volume/parser.go

```diff
@@ -0,0 +1,43 @@
+package volume
+
+import (
+	"runtime"
+
+	"github.com/docker/docker/api/types/mount"
+)
+
+const (
+	// PlatformLinux is the same as runtime.GOOS on linux
+	PlatformLinux = "linux"
```

Comment:
These are operating systems. A platform carries architecture, as well.

We'll have to be careful about how we export this model.

cc @jhowardmsft @estesp

Reply:
Agreed. I suppose volume parsing logic will only be os-specific and architectures should not be taken into account. I'll rename those accordingly.